app/models/post.rb

```diff
@@ -1,6 +1,7 @@
 class Post < ActiveRecord::Base
   # validates :user_id, presence: true
   # validates :content, presence: true
+  scope :newest_first, -> { order(:created_at).reverse_order }
```

Comment:
`order(created_at: :desc)` works and is a bit faster because it actually queries the db to get the order